src/singolo-nft.php

```diff
@@ -104,53 +105,15 @@ function mostraAggiungiRecensione(&$aggiungi_recensione_html, $id) {
         $id=$_GET["id"];
     }
 
-    #Se aggiunge una recensione
-    if (isset($_POST['aggiungi']) && isset($_SESSION['username'])) {
-        $username=$_SESSION['username'];
-        $id=$_POST['id'];
-        $voto=$database->pulisciInput($_POST['voto']);
-        $recensione=$database->pulisciInput($_POST['recensione']);
-
-        $query = "INSERT INTO recensione (timestamp, utente, commento, opera, voto) VALUES (?, ?, ?, ?, ?)";
-        
-        $value = array(date("Y-m-d h:i:s"),$username, $recensione, $id, $voto);
-        $database->executeCRUDPreparedStatement($query,'sssii',$value);
-    }
-
     #Se acquista l'opera
-    if (isset($_POST['acquista']) && isset($_SESSION['username'])) {
-        
-        $username=$_SESSION['username'];
-        //recupera valori del form
-        $id=$_POST['id'];
-        $prezzo=(double)$_POST['prezzo'];
-
-        #controllo che possa acquistare l'opera tramite controllo del saldo
-        $query = "SELECT saldo FROM utente WHERE username = ?";
-        $value=array($username);
-        $saldo = $database->executeSelectPreparedStatement($query,'s',$value);
-        
-        if ($saldo[0]["saldo"]>=$prezzo){
-            $query = "INSERT INTO acquisto (utente, opera, prezzo, data) VALUES (?, ?, ?, ?)";
-            $value = array($username, $id, $prezzo, date("Y-m-d h:i:s"));
-            $database->executeCRUDPreparedStatement($query,'sids',$value);
-
-            #modifico il possessore dell'opera e il saldo dell'utente
-            $query = "UPDATE opera SET possessore = ? WHERE id = ?";
-            $value = array($username, $id);
-            $database->executeCRUDPreparedStatement($query,'si',$value);
-            
-            $query = "UPDATE utente SET saldo = ? WHERE username = ?";
-            $nuovo_saldo=$saldo[0]["saldo"]-$prezzo;
-            $value = array($nuovo_saldo,$username);
-            $database->executeCRUDPreparedStatement($query,'ds',$value);
-            
+    if(isset($_SESSION['acquistato'])){
+        if($_SESSION['acquistato']){
             $acquisto_res.='<p class="center">Opera acquistata con successo!</p>';
         }else{
             $acquisto_res.='<p class="center">Non hai abbastanza ETH per acquistare l\'opera!</p>';
         }
+        unset($_SESSION['risultato_acquisto']);
```

Comment:
Qui credo che dovrebbe essere così, altrimente mostra sempre un messaggio anche quando non serve.
```suggestion
        unset($_SESSION['acquistato']);
```